src/styles/search.css

```diff
@@ -0,0 +1,26 @@
+.search-form {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction` is set to `row` implicitly, you dont need to declare it like this unless you are overwriting a previous definition. Can delete this no problem